sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusSession.cs

```diff
@@ -17,12 +18,32 @@ public class ServiceBusSession
 
         internal string UserSpecifiedSessionId { get; }
 
+        /// <summary>
+        /// Gets the DateTime that the current receiver is locked until.
+        /// </summary>
+        internal DateTime LockedUntilUtcInternal { get; set; }
```

Comment:
Based on discussions from yesterday, this would be guaranteed to be populated on the SessionManager. It should be made public and we can get rid of GetLockedUntilUtcAsync.